schema/geometry_schema.json

```diff
@@ -643,22 +842,79 @@
 						"null"
 					]
 				},
+				"window_definition_type": {
+					"type": "string",
+					"enum": [
+						"Single Window",
+						"Multiple Windows",
+						"Window to Wall Ratio"
+					]
+				},
+				"wwr": {
+					"description": "Target window to wall ratio, only used if window_definition_type is \"Window to Wall Ratio\"",
+					"type": [
+						"number",
+						"null"
+					],
+					"minimum": 0,
+					"maximum": 1,
+					"exclusiveMaximum": true
+				},
 				"height": {
-					"description": "Vertical height of each window. In drawing units.",
-					"type": "number"
+					"description": "Vertical height of each window. In drawing units. Used if window_definition_type is \"Single Window\" or \"Multiple Windows\".",
```

Comment:
One possibility would be to retain these width/height inputs for the WWR option and use them as target values to create multiple windows, evenly spaced and of similar size, that meet the desired WWR.

Reply:
You need a free parameter to get wwr exact, normally we use height as free parameter, sometimes we also have to adjust sill height.  It’s unclear what free parameter would be if you specify height and width and require integer number of windows, plus multiple windows hurt simulation speed and don’t really add any extra accuracy